contracts/helpers/RebalancerFlashloan.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity ^0.8.19;
+
+import "./Rebalancer.sol";
+import { IERC4626 } from "interfaces/external/IERC4626.sol";
+import { IERC3156FlashBorrower } from "oz/interfaces/IERC3156FlashBorrower.sol";
+import { IERC3156FlashLender } from "oz/interfaces/IERC3156FlashLender.sol";
+
+/// @title RebalancerFlashloan
+/// @author Angle Labs, Inc.
+/// @notice Contract built to subsidize rebalances between collateral tokens
+contract RebalancerFlashloan is Rebalancer, IERC3156FlashBorrower {
```

Comment:
Isn't it better to have one contract for all transmuters?

Reply:
That's how we had done it in the first place, given that it's more a utils contract, I'd rather launch it this way for the moment